resources/js/app/helpers/view.js

```diff
@@ -92,6 +92,63 @@ export default {
                 return true;
             },
 
+            checkMimeForUpload(file, objectType) {
+                /** accepted mime types by object type for file upload */
+                const mimes = {
```

Comment:
Maybe as improvement we could define a list of default allowed mime types in configuration and add the list of allowed mime types in `accept` property of  input file 
```html
<input type="file" accept="image/png,image/jpeg,...">
```

Reply:
added a config `uploadMimeTypes`